spec/hardware/system.fmf

```diff
@@ -20,6 +20,9 @@ description: |
            # Integer or string, required number of NUMA nodes.
            numa-nodes: 2|">= 2"
 
+           # System management interface
```

Comment:
Should be `String, system management interface.`

Reply:
@happz addressed